caves.logic

```diff
@@ -0,0 +1,20 @@
+area Caves:
+  room EasternHillsBomb
+  room NorthHyruleFieldCave
+  room NorthHyruleFieldCaveBasement
+  room NorthHyruleFieldFairy
+  room LonLonPuddle
+  room LonLonTingle
+  room LonLonTingleDark
+  room LowerTrilbyAccess
+  room LowerTrilbyAccessScrub
+  room MagicBoomerang
+  room MinishWoodsScrub
+  room SouthHyruleFieldFairy
+  room SouthHyruleFieldPuddle
+  room TownWaterfall
+  room TrilbyDigFairy
+  room TrilbyFairy
+  room TrilbyKeese
+  room TrilbyPuddle
+  room TrilbyScrub
```

Comment:
Isn't this missing the cave where you push the block to access western woods? That's a different (but adjacent) room from the Scrub, right?

Reply:
Oh, those are LowerTrilbyAccess and LowerTrilbyAccessScrub, idk why trilbyscrub is there
Do you have any better name suggestions for them? i was unsure

Reply:
I think I'd prefer keeping them together with the `Trilby` prefix - so keep `TrilbyScrub` and add `TrilbyLowerAccess` or something instead of `LowerTrilbyAccess`?

Reply:
Alright, sounds good